lib/board.rb

```diff
@@ -1,27 +1,31 @@
 require './lib/cell'
+require 'pry'
 
 class Board
   attr_reader :cells
 
-  def initialize(range = "A1".."D4")
+  def initialize(range)
+    @max_board_size = 0
     @cells = populate_with_cells(range)
-    @max_board_size = 4
   end
 
   def populate_with_cells(range)
-    @max_board_size = range.max_by { |coords| coords }[1].to_i
-
+    @max_board_size = range.last.scan(/\d+/).join.to_i
```

Comment:
### `scan` 
 - finds something that matches what is passed in its args.
 - `/\d+/` is a regex for finding a number
 - `join` to merge the array with one match to a string

Reply:
Perfect! Regex is so efficient! 🤙